gluster-exporter/metric_peer_counts.go

```diff
@@ -0,0 +1,224 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"io"
+	"os/exec"
+	"strconv"
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus"
+	log "github.com/sirupsen/logrus"
+)
+
+// executes the command 'cmdStr'
+// returns the byte array as output or error
+func execCommand(cmdStr string) ([]byte, error) {
```

Comment:
there is already one execute util function. can that be modified? https://github.com/gluster/gluster-prometheus/blob/master/pkg/glusterutils/exporterd.go#L138

Reply:
Had to make the function (executeCmd) to a publicly available one (ExecuteCmd) and changed all the other places where it's being called. Hope that is fine.